sdcm/nemesis.py

```diff
@@ -1757,7 +1757,7 @@ def _mgmt_backup(self, backup_specific_tables):
         status = mgr_task.wait_and_get_final_status(timeout=54000, step=5, only_final=True)
         if status == TaskStatus.DONE:
             self.log.info("Task: %s is done.", mgr_task.id)
-        elif status == TaskStatus.ERROR:
+        elif status in (TaskStatus.ERROR, TaskStatus.ERROR_FINAL):
```

Comment:
It is imposible to get TaskStatus.ERROR out of `wait_and_get_final_status` with this parameters, so please drop it

Reply:
Techincally if the final status is `ERROR (x/4)` where x is not 4, it will be counted as `ERROR`. Or if we run a task with no retry, it will end in just an `ERROR`
Sure, it's highly unlikely, but I'd say it's better safe than sorry.

Reply:
will this `ERROR` be accounted as final status (as you set `only_final=True`)?
or it will be the final status only when `ERROR (x/4)` where `x == 4`?

Reply:
it is not only accounted there is only one final `ERROR` status, which is `ERROR (4/4)`

Reply:
so, when `ERROR (2/4)` it won't be returned from `wait_and_get_final_status` with param `only_final=True`, right?